### DISCUSSION TITLE
How do I disable the YouTube permanent player control?

### DISCUSSION THREAD

**Discussion by LoonyBeLuna** [ReVanced/revanced-discussions#1033] [Question] [answered]
Hi, I just got ReVanced and installed the better YouTube, but when I open a video it shows me this.
![Screenshot_20240328_010939_YouTube](https://github.com/ReVanced/revanced-discussions/assets/131197997/c2cfd185-c766-43fa-9ab6-77bde7b5b0d6)

I have to click the X at the right top to close it.
How can I disable it?
I guess it's part of one of the patches?
I installed the recommended YouTube app with 60 patches.

I hope someone can help me.

**Comment by oSumAtrIX** [comment, marked as answer]
That's an accessibility setting you have manually enabled on YouTube

**Reply by LoonyBeLuna** [reply]
Hi, I never had that in the official YouTube app.

**Reply by LoonyBeLuna** [reply]
I found it, you are right, I DID enable it..

I'm so dumb 😑🙃
Thank you for the fast answer!
